tajo-cluster-tests/src/test/java/org/apache/tajo/QueryTestCaseBase.java

```diff
@@ -913,7 +913,8 @@ private Path getDataSetFile(String fileName) throws IOException {
           throw new IOException("Cannot find " + fileName + " at " + currentQueryPath + " and " + namedQueryPath);
         }
       } else {
-        throw new IOException("Cannot find " + fileName + " at " + currentQueryPath + " and " + namedQueryPath);
+        // to make empty table (used to insert test)
```

Comment:
This code will make a directory in a resource directory. It seems to be not good. Could you remove this change?